src/(components)/ForgetPasswordForm.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+
+interface FormData {
+  email: string
+}
+
+const ForgotPasswordForm = () => {
+  // Use the useState hook to manage the form data
+  const [formData, setFormData] = useState<FormData>({
```

Comment:
A better way would be just to use handleSubmit rather than add a hook to store the data in the component - the html <form> will submit the data to the function directly